app/views/providers/limitations/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= page_template(
+      page_title: t('.h1-heading')
+    ) %>
+
+<% if @legal_aid_application.proceeding_types.present? %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-heading-m">
+        <%= t('.h2-heading', count: pluralize(@legal_aid_application.proceeding_types.size, t('.h2-heading'))) %>
+      </h2>
+      <div class="selected-proceeding-types">
+        <%= render partial: 'shared/forms/proceedings_types/proceeding_type', collection: @legal_aid_application.proceeding_types, locals: { translation_path: 'providers.limitations.show' } %>
+      </div>
+
+      <div class="govuk-!-padding-bottom-6"></div>
+
+      <p class="govuk-body">
+        <%= t('.form_of_service.html') %> <%= t('.form_of_service_type') %> <%= content_tag(:br) %>
+        <%= t('.incurrable_costs.html') %> <%= t('.incurrable_costs_value') %> <%= content_tag(:br) %>
```

Comment:
Using `<br>` for spacing is not a great practice I believe.
Furthermore, `content_tag(:br)` generates two `br` tags like that: `<br></br>` (which is kind of hacky).
I'd suggest using spacing like `<div class="govuk-!-padding-bottom-6"></div>` instead.

I was trying a few things and this doesn't need extra spacing and looks very close to the prototype.
```html
      <div class="govuk-body">
        <p><%= t('.form_of_service.html') %> <%= t('.form_of_service_type') %></p>
        <p><%= t('.incurrable_costs.html') %> <%= t('.incurrable_costs_value') %></p>
        <p><%= t('.allowable_work.html') %> <%= t('.body') %></p>
      </div>
```

You could add a bit of spacing between the `p` tags:
```html
      <div class="govuk-body">
        <p><%= t('.form_of_service.html') %> <%= t('.form_of_service_type') %></p>
        <div class="govuk-!-padding-bottom-1"></div>
        <p><%= t('.incurrable_costs.html') %> <%= t('.incurrable_costs_value') %></p>
        <div class="govuk-!-padding-bottom-1"></div>
        <p><%= t('.allowable_work.html') %> <%= t('.body') %></p>
      </div>
```

Reply:
If you go for Julien's paragraph markup, I'd suggest either:
```erb
<p><strong><%= t('.form_of_service.html') %></strong> <%= t('.form_of_service_type') %></p>
```
Or use spans and style them with CSS rather than having the styling defined in the translation file.

Reply:
I used <strong> with Julians suggestion. it seemed a bit much to add css just to bold these items.